Source/StringScore.swift

```diff
@@ -117,13 +117,13 @@ public extension String {
       }
       
       // Same case bonus.
-      if (string[idxOf] == word[word.index(word.startIndex, offsetBy: i)]) {
+      if (lString[idxOf] == word[word.index(word.startIndex, offsetBy: i)]) {
```

Comment:
does this defeat the same case bonus? (lString being always lower cased?)

Reply:
Hey, sorry for the delay. You may be right, to be honest its been a while so not 100% sure. It sure looks like at face value that we would miss the bonus.

In that case the other solution would be to replace line 60 with `let string = self + ""` which would ensure the 'string' is a swift string as opposed to autoboxed NSString. That should then keep the same case bonus as far as I know